Lesson10/Task12.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Сделайте поле ввода, в которое пользователь вводит год (4 цифры), а скрипт определяет високосный ли год.
+ */
+if (
```

Comment:
What if condition is false?

Reply:
If condition is false, it just prints the form.